terraso_backend/apps/story_map/views.py

```diff
@@ -52,13 +52,28 @@ def post(self, request, **kwargs):
         )
 
         entry_form = StoryMapForm(data=form_data)
-
         if not entry_form.is_valid():
             error_messages = get_error_messages(entry_form.errors.as_data())
             return JsonResponse(
                 {"errors": [{"message": [asdict(e) for e in error_messages]}]}, status=400
             )
 
+        if "chapters" in config:
+            for chapter in config["chapters"]:
+                media = chapter.get("media")
+                if not (
+                    media
+                    and (media["type"].startswith("image")
+                         or media["type"].startswith("audio")
+                         or media["type"].startswith("video"))
+                ):
+                    logger.info("Warning: invalid media type")
+                    error_message = ErrorMessage(
+                        code="Invalid Media Type",
+                        context=ErrorContext(model="StoryMap", field=NON_FIELD_ERRORS)
+                    )
+                    return JsonResponse({"errors": [{
+                        "message": [asdict(error_message)]}]}, status=400)
```

Comment:
Question: Did you check how this gets to the client, I think ideally the code should be something that we can understand form the code side, because it should be localized in the client side

Reply:
Invalid media type upload request will be rejected on the backend and a warning will be logged to console